docs/reference/rosbridge.md

```diff
@@ -78,6 +78,46 @@ Parameters:
 
 - **message**: *engine.target.Pose*  
   OpenDR pose to be converted to ROS pose.
+ 
+#### `ROSBridge.from_ros_face`
+
+```python
+ROSBridge.from_ros_face(self,
+                        ros_hypothesis)
+```
+
+Converts a ROS ObjectHypothesis message into an OpenDR Category.
+
+Parameters:
+
+- **message**: *ros_bridge.msg.ObjectHypothesis*  
+  ROS ObjectHypothesis to be converted into an OpenDR Category.
+  
+#### `ROSBridge.to_ros_face`
```

Comment:
The name and description of the functions `to_ros_face)` and `from_ros_face()` do not match. If these functions provide the generic opendr category-> ros functionality, then they should be probably renamed to `to_ros_category()` and `from_ros_category()`. On the other hand, if they provide face specific functionality, their description should be narrowed down to indicate this fact.

Reply:
They are specific in the term that OpenDR.Category holds 3 Variables (data, description, confidence) which are broken down to ObjectHypothesis for data and confidence and String for description. Thus the naming, but if a more proper naming comes to mind, please propose it.